src/modules/clark/learning-object-module/learning-objects.routes.ts

```diff
@@ -318,7 +318,9 @@ export const LEARNING_OBJECTS_ROUTES: ProxyRoute[] = [
      */
     {
         method: HTTPMethod.GET,
-        path: "/users/:userId/learning-objects/:learningObjectId/submissions",
+        path: "/learning-objects/:learningObjectId/submissions",
+        target: envConfig.getUri(CLARK_SERVICE_URI),
+        auth: true,
```

Comment:
I thought about leaving this comment on your clark-service PR but double check authentication is required. We would have seen `auth:true` before but it looks like you added

Reply:
mike and i talked, resolved